commit 1dfce1bf8cdd5855fdb3be5c09c510206dce436a
Author: Rolf Eike Beer <kde@opensource.sf-tec.de>
Date:   Tue Jan 15 15:05:56 2013 +0100

    make sure all members are properly initialized

diff --git a/model/selectkeyproxymodel.cpp b/model/selectkeyproxymodel.cpp
index 47b56aa..92073df 100644
--- a/model/selectkeyproxymodel.cpp
+++ b/model/selectkeyproxymodel.cpp
@@ -1,4 +1,4 @@
-/* Copyright 2008  Rolf Eike Beer <kde@opensource.sf-tec.de>
+/* Copyright 2008,2010,2012,2013  Rolf Eike Beer <kde@opensource.sf-tec.de>
  *
  * This program is free software; you can redistribute it and/or
  * modify it under the terms of the GNU General Public License as
@@ -25,7 +25,9 @@
 using namespace KgpgCore;
 
 SelectKeyProxyModel::SelectKeyProxyModel(QObject *parent)
-	: QSortFilterProxyModel(parent), m_model(NULL)
+	: QSortFilterProxyModel(parent),
+	m_model(NULL),
+	m_showUntrusted(false)
 {
 }
 
